exotations/task_maps/task_map/src/Point2Line.cpp

```diff
@@ -47,7 +47,7 @@ Eigen::Vector3d Point2Line::distance(const Eigen::Vector3d &point, const Eigen::
     // t = ((s-p)*(e-s)) / (|e-s|^2)    (* denotes the dot product)
 
```

Comment:
This function does not return the point to line distance, it returns the vector from the closes point on the line towards the  query point. The name of the function does not reflect this.